.gitignore

```diff
@@ -45,3 +45,6 @@ workers/unity/Assets/Improbable.Generated.NetworkTypes/Generated/**/*.meta
 /workers/unity/Assets/Improbable.Generated.NetworkTypes/Generated/improbable/gdk/tests/*.cs
 
 code_generator/GdkCodeGenerator/Generated/
+
+# Schema from Unity Package Manager Packages (Copied via tools/CopySchema)
+schema/from_gdk_packages
```

Comment:
Generated code doesn't car about which directory this is in does it? I thought the schema compiler used to care, but I might have just been a dream.

Reply:
Spatial cares when it generates the schema descriptor

Reply:
Sorry I meant it doesn't do something weird like require packages to be in the corresponding directory? If this has worked then the answer is probably no.